data/campaigns/Heir_To_The_Throne/scenarios/15_The_Lost_General.cfg

```diff
@@ -347,10 +347,53 @@
             y=1-23
             side=1
         [/filter]
+
+        [sound]
+             name=thunderstick.ogg
+        [/sound]
+        [delay]
+            time=200
+        [/delay]
+
+        # This dialog has several purposes:
+        # 1. Clue to the player that he's exploring in the right direction
+        # 2. Clue to the player to hurry up, in order to save Geldar
+        # 3. An excuse to play the thunderstick sound :)
         [message]
-            speaker=unit
-            message= _ "These passages seem to have been used recently."
+            speaker=Konrad
+            # We don't use {WHISPER} since it adds parentheses and two pairs of parentheses would be confusing.
+            message=_ "<i>(motions for silence)</i>
+
+<small>What was that raucous noise?</small>"
+        [/message]
+        [message]
+            speaker=Delfador
+            {DELFADOR_MENTORING}
+            message=_ "That can only have been a Dwarven thunderstick, Konrad. Our Knalgan allies must be in the thick of battle."
         [/message]
+        # Let $unit speak too, so it's clear where the battle is.
+        [if]
+            [have_unit]
+                id=$unit.id
+                race=dwarf
+            [/have_unit]
+            [then]
+                [message]
+                    speaker=unit
+                    # po: speaker is a dwarf
+                    message=_ "That’ll be Lord Geldar’s clan, my lords. Their scouting patrols should’ve found us by now."
+                [/message]
+            [/then]
+            [else]
+                [message]
+                    speaker=unit
+                    # po: The unit is saying that a battle between the player's allies and enemies is not far, but still under the shroud. Translate freely.
+                    male_message=_ "It’s not far now, my lords."
+                    # po: The unit is saying that a battle between the player's allies and enemies is not far, but still under the shroud. Translate freely.
+                    female_message=_ "female^It’s not far now, my lords."
```

Comment:
This is likely to come from a unit that has never been in these tunnels before, yet the phrase "it's not far now" implies some familiarity. I suggest either dropping this `[else]`, or having the unit say that the sound definitely came from in front of them.

Reply:
As the comment says, the [else] has a purpose: to draw the player's attention to the part of the map where he's about to discover the allies. The line is a bit tricky to write because it can be triggered by units going either through the north of the map, or through the south ("Guest Quarters"). How about, say, a simple `We’ll proceed carefully, my lords`? Or perhaps `We must hurry to assist them, my lords`, if we want to steer the player to saving Geldar? Any other ideas?

Reply:
Oh I see, reading it again now it sounds like the non-dwarf speaker could be judging distance on the basis of how loud the thunderstick sound was. Regardless, the latter (hurry to assist) sounds like a good option.